folium/map.py

```diff
@@ -22,6 +22,10 @@
 
 ENV = Environment(loader=PackageLoader('folium', 'templates'))
 
+_d3_js = [
+    ('d3', 'http://d3js.org/d3.v4.min.js'),
```

Comment:
This should be in the plugin and not in the global map.

Reply:
Alright.